src/azure-cli/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+**Policy**
+
+* Support for Policy new API version 2019-06-01
```

Comment:
Because you also upgrade the azure-mgmt-resource package to 4.0. Please declare the version change in HISTORY.rst.